modules/deployment-scripts/create-kv-certificate/create-kv-cert.sh

```diff
@@ -12,7 +12,7 @@
 
     if [ -z "$issuerName" ] || [ -z "$issuerProvider" ]; then
         policy=$(az keyvault certificate get-default-policy \
-            | sed -e s/CN=CLIGetDefaultPolicy/CN=${certCommonName}/g )
+            | sed -e s/CN=CLIGetDefaultPolicy/CN="${certCommonName}"/g )
     else
       if [ "$issuerProvider" == "DigiCert" ] || [ "$issuerProvider" == "GlobalCert"]; then
```

Comment:
🚫 **[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Couldn't parse this test expression. Fix to allow more checks. [SC1073](https://github.com/koalaman/shellcheck/wiki/SC1073)